src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,12 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(str -> str.split(","))
+                .flatMap(Arrays::stream)
+                .map(Integer::parseInt).filter(num -> num % 2 == 0)
+                .min(Integer::compareTo)
```

Comment:
```suggestion
                .min()
```
It’s  default behavior

Reply:
fix comment